oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticBulkProcessorHandler.java

```diff
@@ -151,6 +153,7 @@ private BulkProcessor initBulkProcessor() {
         return BulkProcessor.builder(requestConsumer(),
                 new OakBulkProcessorListener(), this.indexName + "-bulk-processor")
                 .setBulkActions(indexDefinition.bulkActions)
+                .setConcurrentRequests(BULK_PROCESSOR_CONCURRENCY)
```

Comment:
@amit-jain : Can the concurrentRequests property be moved to index definition, similar to other properties like bulkSize, which would enable the caller to configure appropriate concurrentRequests in elastic index definition.

Reply:
This config does not seem suitable to be included in the IndexDefinition as it is affecting the processing in general and not per index. @nit0906 wdyt?

Reply:
@amit-jain  - agreed.

the bulkProcessor instance is actually linked to index definition and all other configurations like bulk size threshold , flush interval etc are also defined/configured at index definition level. So if we want to keep configurations in the same line - then it would seem the concurrency can also go in the index definition. 

But this is a different case where we would want to distinguish this on the basis of whether indexing is running from oak-run or otherwise - so configuring this in index definition won't be possible (since the index def would remain the same in both cases but we would want the concurrency to be different - so a system property is the only solution here.)